pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/BaseContext.java

```diff
@@ -201,4 +202,14 @@
     default PulsarClient getPulsarClient() {
         throw new UnsupportedOperationException("not implemented");
     }
+
+    /**
+     * Get the pulsar client builder.
```

Comment:
can we add something like "This Builder is pre-configured with Connection parameters, Authentication credentials and other context values.
You can use this Builder in order to connect to the local Pulsar cluster.
The Function will be responsible for disposing any PulsarClient created by this builder."

Reply:
After a second thought, I think User should be responsible for cleaning any client created by using the ClientBuilder.

The Function Runtime is only responsible for closing the client get via `getPulsarClient()`.